scalalib/src/Dependency.scala

```diff
@@ -18,6 +18,10 @@ object Dependency extends ExternalModule {
         allowPreRelease)
     }
 
+  def showUpdates(ev: Evaluator, allowPreRelease: Boolean = false) = T.command{
```

Comment:
Can you please add some ScalaDoc to the new command. It would be great, if you could also update/add the comment of the changed `updates` command.